Source/CsDebugScript.Engine/Engine/Utility/SimpleCache.cs

```diff
@@ -28,13 +64,28 @@ public static SimpleCacheStruct<T> CreateStruct<T>(Func<T> populateAction)
         {
             return new SimpleCacheStruct<T>(populateAction);
         }
+
+        /// <summary>
+        /// Invalidates all the instances of type <see cref="SimpleCacheStruct{T}" /> and
+        /// <see cref="SimpleCache{T}" /> which are fields of class given as root object and any
+        /// fields of the same type in child fields recursively.
+        /// Use it when there are massive changes and all the caches need to be invalidated.
+        /// </summary>
+        /// <typeparam name="T">Type of the root object.</typeparam>
+        /// <param name="rootObject">Root object from which we recursively drop the caches.</param>
+        public static void InvalidateCacheRecursively<T>(T rootObject)
```

Comment:
Nope, missed leftover. Removed.

---
In reply to: [70602613](https://github.com/southpolenator/WinDbgCs/pull/8#discussion_r70602613) [](ancestors = 70602613)